tests/testsuite/install_upgrade.rs

```diff
@@ -854,7 +854,7 @@ fn partially_already_installed_does_one_update() {
 [FINISHED] release [optimized] target(s) in [..]
 [INSTALLING] [CWD]/home/.cargo/bin/baz[EXE]
 [INSTALLED] package `baz v1.0.0` (executable `baz[EXE]`)
-[SUMMARY] Successfully installed foo, bar, baz!
+[SUMMARY] Successfully installed foo, bar and baz!
```

Comment:
Looking at the test output, I feel like using `and` makes it harder to quickly scan.  Using an oxford comma would help but that doesn't help with the `foo and bar` (2 item) case.